src/python/pants/bin/goal_runner.py

```diff
@@ -214,7 +214,10 @@ def __init__(self, context, goals, run_tracker, kill_nailguns, exiter=sys.exit):
 
   @classmethod
   def subsystems(cls):
-    """Subsystems used outside of any task."""
+    """Subsystems used outside of any task.
+
+    TODO: Should these simply be dependencies of the GlobalOptions optionable?
```

Comment:
Placing them somewhere more specific rather than less would be my preference... so here is better for now.

Reply:
Fair, I guess we don't want GlobalOptions to have all those dependencies. Will remove the TODO.